shared/utils/enable-derivatives.py

```diff
@@ -221,14 +221,24 @@ def main():
     root = tree.getroot()
 
     benchmarks = []
-    benchmarks.extend([
-        (XCCDF11_NS, elem)
-        for elem in list(root.findall(".//{%s}Benchmark" % (XCCDF11_NS)))
-    ])
-    benchmarks.extend([
-        (XCCDF12_NS, elem)
-        for elem in list(root.findall(".//{%s}Benchmark" % (XCCDF12_NS)))
-    ])
+
+    def scrape_benchmarks(root_element, namespace, dest):
+        dest.extend([
+            (namespace, elem)
+            for elem in list(root.findall(".//{%s}Benchmark" % (namespace)))
+        ])
+        if root_element.tag == "{%s}Benchmark" % (namespace):
+            dest.append((namespace, root_element))
+
+    scrape_benchmarks(
+        root, XCCDF11_NS, benchmarks
```

Comment:
I am not experienced python programmer, but this syntax looks very strange. I think it's function call, but I am not sure at first glance.